tests/test_ms2deep_client.py

```diff
@@ -0,0 +1,94 @@
+import pytest
+import requests
+
+from omigami.ms2deep import MS2Deep, InvalidNumberOfSpectra
+from omigami.spec2vec import InvalidCredentials
+
+
+def test_build_payload(mgf_generator):
+    client = MS2Deep("token")
+
+    payload = client._build_payload((mgf_generator[:2]))
+
+    assert "data" in payload.keys()
+    assert len(payload["data"]["ndarray"]["data"]) == 2
+
+
+@pytest.mark.skip(reason="This test should only work when the endpoint is working")
+@pytest.mark.internet_connection
+def test_send_request():
+    client = MS2Deep("bad_token")
+    small_payload = {
+        "data": {
+            "ndarray": {
+                "parameters": [],
+                "data": [
+                    {
+                        "peaks_json": "[[80.060677, 157.0], [337.508301, 230.0]]",
```

Comment:
Maybe also change it here just for consistency, even though I know in this case it doesn't matter because we are testing the bad token :))